includes/rest-api/class-sensei-rest-api-lessons-controller.php

```diff
@@ -139,4 +140,13 @@ protected function add_additional_fields_to_object( $prepared, $request ) {
 
 		return $prepared;
 	}
+
+	/**
+	 * Handle meta args.
+	 *
+	 * @param array $args The args.
+	 */
+	public function filter_lesson_meta_args( $args ) {
```

Comment:
I think this filter isn't doing anything, right? Or am I missing something?

Reply:
Yeah. This one is redundant. It's a leftover from my first approach for determining the preview/customize urls for learning mode. Removed! 👍